eva/parser/lark_visitor/_functions.py

```diff
@@ -125,22 +136,24 @@ def get_aggregate_function_type(self, agg_func_name):
         return agg_func_type
 
     def aggregate_windowed_function(self, tree):
-        agg_func_name = self.visit(tree.children[0]).value
+
         agg_func_arg = None
-        assert agg_func_name in [
-            "MIN",
-            "MAX",
-            "AVG",
-            "SUM",
-            "COUNT",
-            "FIRST",
-            "LAST",
-            "SEGMENT",
-        ]
+        agg_func_name = None
+
         for child in tree.children:
             if isinstance(child, Tree):
                 if child.data == "function_arg":
                     agg_func_arg = self.visit(child)
+                elif child.data == "aggregate_function_name":
+                    agg_func_name = self.visit(child).value
+            elif isinstance(child, Token):
+                token = child.value
+                # Support for COUNT(*)
```

Comment:
I don't understand this logic. Are we hardcoding `*` to `id` in the parser? If yes, I guess even though it is hacky, it saves us from handling this corner case in the binder. We could change it to `IDENTIFIER_COLUMN`, which is supposed to be a unique row id in all the tables.

Reply:
Yes. This query currently works. I was also worried if "id" will be always present. How should I change it to `IDENTIFIER_COLUMN`?

Reply:
The unit tests do not create tables using `IDENTIFIER_COLUMN` -- so the test case fails.

Reply:
Yeah, I verified we don't support projecting `IDENTIFIER_COLUMN`, which causes the binder to fail.
`id` won't work for images or other tables. Ideally, the binder should take care of it. 
An `if` condition  [here](https://github.com/georgia-tech-db/eva/blob/04368ef72a04973b5ad3bfdf98b34e957440539e/eva/binder/statement_binder.py#L103) should fix it.

Reply:
Reference: https://github.com/georgia-tech-db/eva/blob/04368ef72a04973b5ad3bfdf98b34e957440539e/eva/binder/binder_utils.py#L77